perma_web/functional_tests/tests.py

```diff
@@ -283,7 +325,7 @@ def repeat_while_false(func, timeout=10, sleep_time=.1):
                 time.sleep(sleep_time)
 
         def fix_host(url):
```

Comment:
Is this method needed any more?

Reply:
Actually.... this could easily be why the functional tests are failing on travis if I inspect the contents of the iframe directly. Is the iframe's src pointed at the test server, for pywb playback? How could it be? (Why does it work locally.....)

Reply:
Putting back the iframeless loading of the warc causes tests to pass, which is good enough. But, experimenting w/ a tweaked `/etc/hosts` on travis and direct inspection of the iframe, because that would be better.

Reply:
YES! It was just `/etc/hosts`